control/summary_table_test.go

```diff
@@ -36,6 +36,13 @@ func docFixture(control string) *xml.XmlDocument {
 	return doc
 }
 
+func getTable(control string) xml.Node {
+	doc := docFixture(control)
+	tables, err := doc.Search("//w:tbl")
```

Comment:
The problem is `docFixture()` is loading from `simplified_table.xml` rather than a full Word doc, and we can't construct a full `Document` from that. Let me know what you think of my workaround.